src/usr/local/www/system_advanced_firewall.php

```diff
@@ -569,6 +576,13 @@
 	'connected networks and VPN networks when using policy routing. This can be disabled '.
 	'for special purposes but it requires manually creating rules for these networks.');
 
+$section->addInput(new Form_Checkbox(
+	'no_apipa_block',
+	'Allow APIPA',
+	'Allow APIPA traffic',
+	$pconfig['no_apipa_block']
+))->setHelp('Normally this traffic is dropped by firewall (as APIPA traffic cannot be routed), but some providers may utilize APIPA space for interconnect interfaces.');
```

Comment:
Should be `by the firewall`, also you can remove the `()` around the last part of that phrase.

Reply:
done